README.md

```diff
@@ -203,3 +144,20 @@ npm ERR! node-libcurl@1.3.3 install: `node-pre-gyp install --fallback-to-build`
 ```
 
 It is probably related to a gcc library that is failing with some versions of Node and npm, try with Node 10 without updating the `npm` version that comes with it.
+
+## Tests
+
+Run tests with docker-compose:
+
+```bash
+docker-compose  -f docker-compose.test.yml build cht-couch2pg
+docker-compose  -f docker-compose.test.yml run cht-couch2pg grunt test
+```
+
+Run tests in interactive watch mode with: `docker-compose -f docker-compose.test.yml run cht-couch2pg npm run watch`
```

Comment:
This fails for me. 

```.shell
❯ docker-compose -f docker-compose.test.yml run cht-couch2pg npm run watch
Starting cht-couch2pg_postgres_1 ... done
Starting cht-couch2pg_couch_1    ... done

> cht-couch2pg@3.4.0 watch /app
> npm run jest --projects jest-*.config.js --watchAll

npm ERR! missing script: jest
```

I think it is becauase the `watch` script in the package.json is `npm run jest --projects jest-*.config.js --watchAll` when I think it should just be `jest --projects jest-*.config.js --watchAll`.

Reply:
I think you're right! Fixed in 911e73f

i think also you caught a bug that has existed for a while before this PR ;)